golos.publication/golos.publication.hpp

```diff
@@ -45,7 +47,7 @@ class publication : public contract {
     static fixp_t get_delta(atmsp::machine<fixp_t>& machine, fixp_t old_val, fixp_t new_val,
         const structures::funcinfo& func);
 
-    static void check_upvote_time(uint64_t cur_time, uint64_t mssg_date);
+    static void check_upvote_time(uint64_t cur_time, uint64_t mssg_date, name code);
```

Comment:
btw, why do you need to have this `static`? it's `private`, so can't be used outside. propose remove `static` and `name code`, because `_self` will be accessible